minimesos/src/main/java/com/containersol/minimesos/cluster/MesosCluster.java

```diff
@@ -41,6 +43,8 @@
 
     public static final String MINIMESOS_HOST_DIR_PROPERTY = "minimesos.host.dir";
 
+    public static final String SLAVE_MAPPING_DIR = "agent-uuid-";
+
     private static DockerClient dockerClient = DockerClientFactory.build();
```

Comment:
I don't think it's necessary to create separate folders for Mesos Agents. Per cluster we can have a single `sandbox` folder because within that each agent will write to its own folder. Also refactor `SLAVE` to `AGENT`.